### PR TITLE
got rid of most bugs

### PR DESCRIPTION
**Pull request by kac320**

### CODE REVIEW

File: src/codeu/chat/common/User.java [hunk 1]
```diff
@@ -66,21 +66,36 @@ public User(Uuid id, String name, Time creation) {
...
+    System.out.println("HERE! -------------------------" + followees.size());
+    System.out.println("HERE! -------------------------" + followers.size());
+    System.out.println(this);
+    for (Uuid user : followees.keySet()){
+      status.append(followees.get(user).statusUpdate());
+    }
     return status.toString();
   }
 
+  // // User A wants to follow User B
```
Comment (turtlefarmer) [inline src/codeu/chat/common/User.java]: delete unused code instead of commenting out.

File: src/codeu/chat/common/User.java [hunk 2]
```diff
@@ -66,21 +66,36 @@ public User(Uuid id, String name, Time creation) {
 
   public String statusUpdate() {
     StringBuilder status = new StringBuilder();
-    for (Uuid user : following.keySet())
-      status.append(following.get(user).statusUpdate());
+    System.out.println("HERE! -------------------------" + followees.size());
```
Comment (turtlefarmer) [inline src/codeu/chat/common/User.java]: Either delete or cleanup debugging code.

File: src/codeu/chat/server/Model.java [hunk 3]
```diff
@@ -104,9 +104,10 @@ public void unfollowUser(User userA, User userB) {
   }
 
   public void followUser(User userA, User userB) {
-    User user1 = userById().first(userA.id);
-    User user2 = userById().first(userB.id);
-    User.follow(user1, user2);
+    // User user1 = userById().first(userA.id);
```
Comment (turtlefarmer) [inline src/codeu/chat/server/Model.java]: delete commented out code.

**Review by turtlefarmer — CHANGES_REQUESTED** [review]

File: src/codeu/chat/common/User.java (hunk 2 above)
Comment (kac320) [reply src/codeu/chat/common/User.java]: not quite sure what you mean here. which lines should I delete?

File: src/codeu/chat/server/Model.java (hunk 3 above)
Comment (kac320) [reply src/codeu/chat/server/Model.java]: done

**Comment by turtlefarmer** [comment]
Please resolve conflicts and I'll take another look.

**Pull request merged**